source/54-1 Timothy.usfm.db

```diff
@@ -156,8 +156,8 @@ This must be our testimony, as opportunities present themselves;
 \v 1 All who are in the position of slaves should regard their masters as deserving of the greatest respect, so that the name of God, and our teaching, may not be maligned.
 \v 2 Those who have Christian masters should not think less of them because they are [neut:also followers of Christ|masc:their brothers], but on the contrary they should serve them all the better, because those who are to benefit by their good work are dear to them as their fellow Christians.
 \s Conclusion
-Those are the things to insist on in your teaching.
 \p
+Those are the things to insist on in your teaching.
```

Comment:
Looks like there are several other cases with different whitespace formatting. 66 matches for "\\p\s+\w".